docs/api/instruments/jobinyvon/jobinyvon270m.rst

```diff
@@ -0,0 +1,12 @@
+######################
+Jobin Yvon Instruments
+######################
+
+
+Spectro270M Spectrometer
+========================
```

Comment:
This file is only for the Spectro 270M spectrometer, therefore you do not need to repeat the manufacturer name. The manufacturer name from `jobinyvon/index.rst` will be the header!

```suggestion
########################
Spectro270M Spectrometer
########################
```